imagestreams/jupyter-rust/imagestream.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: jupyter-rust
+  namespace: redhat-ods-applications
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: >-
+      https://github.com/nerc-images/jupyter-rust
+    opendatahub.io/notebook-image-name: >-
+      Jupyter Rust
+    opendatahub.io/notebook-image-desc: >-
```

Comment:
I think you probably want a literal block here (`|`) instead of a folding block `>`, because a folding block will remove all your newlines, so you end up with:

```
An OpenShift AI Image running Jupyter Lab for Rust development. - Based on the [Evcxr project](https://github.com/evcxr/evcxr) on GitHub for Jupyter Lab Notebook integration. - Uses Rust 1.79. - Used by Professor Leonidas Kontothanassis, MassMutual Professor of the Practice of Computing & Data Sciences + Director of Industry Engagement.
```

Instead of:

```
An OpenShift AI Image running Jupyter Lab for Rust development.
- Based on the [Evcxr project](https://github.com/evcxr/evcxr)
on GitHub for Jupyter Lab Notebook integration.
- Uses Rust 1.79.
- Used by Professor Leonidas Kontothanassis, MassMutual Professor of the
Practice of Computing & Data Sciences + Director of Industry Engagement.
```

Reply:
Thanks for the suggestion @larsks , I updated a couple more of the imagestream descriptions. Yes @schwesig , I just wanted to list them alphabetically.